airflow/config_templates/config.yml

```diff
@@ -635,7 +635,7 @@
       default: "%%(asctime)s %%(levelname)s - %%(message)s"
     - name: log_formatter_class
       description: ~
-      version_added: 2.3.3
+      version_added: 2.3.4
```

Comment:
bump up.
#24373 didn't show up on https://github.com/apache/airflow/pull/24762/files#diff-9b07fd28838e759e05bdaa5293886d860374fe291bdc6cc43c20f951fc77eab4 and it's commits